Makefile

```diff
@@ -23,7 +23,7 @@ SHELL:=/usr/bin/env bash
 #
 # Go.
 #
-GO_VERSION ?= 1.17.3
+GO_VERSION ?= 1.18.3
```

Comment:
There's another GO_VERSION in test/infrastructure/docker/Makefile

Reply:
Done!